### PR TITLE
#166789859 Configure postgres database using sequelize

### PR DESCRIPTION
**Pull request by adafia**
#### What does this PR do?
This pull request adds PostgreSQL configuration with `Sequelize`. 

#### Description of Task to be completed?
The task was to switch out the existing database connection and configuration with PostgreSQL using `Sequelize`.

#### How should this be manually tested?

> install dependencies and development dependencies
```
> npm install
```

> create database to postgresql
```
> createdb authorsHavenDb
```
remember to change the password in config.json file for development

> create your config file
```
> create config.json file into 'config/' and add same data as 'example.config.json' 
remember to change database user, and password
```

> add table to database

```
> sequelize db:migrate
```
The above command will migrate all table to your database

#### Any background context you want to provide?
The initial codebase was setup with MongoDb

#### What are the relevant pivotal tracker stories?
[#166789859](https://www.pivotaltracker.com/story/show/166789859)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by Emile-Nsengimana — COMMENTED** [review]
- I think that description of Task to be completed has to be there as long as you are raising a PR for a finished task?
- and also it will be better if we use a .env file for our configuration so that each one of us doesn't have to make unnecessary changement

**Review by j4l13n — APPROVED** [review]

**Review by manzif — APPROVED** [review]

**Review by Dom58 — COMMENTED** [review]
Please, leave a new blank line to every file

**Review by emukungu — CHANGES_REQUESTED** [review]
use a .env file for the config as requested by @Emile-Nsengimana for the whole project.

File: config/config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        username: "root",
+        password: null,
+        database: "database_production",
+        host: "127.0.0.1",
+        dialect: "postgres",
+        operatorsAliases: false
+    }
+}
+
+module.exports = config;
```
Comment (emukungu) [inline config/config.js]: add a new line

**Pull request merged**